PostBy/Views/MapPin.m

```diff
@@ -31,7 +31,7 @@ + (MapPin *)createPinFromPostVM:(PostViewModel *)postVM {
     pin.postVM = postVM;
     
     // Default every pin's image to be 50 x 50
-    CGSize pinImgSize = CGSizeMake(50, 50);
+    CGSize pinImgSize = CGSizeMake(PIN_IMG_SIZE, PIN_IMG_SIZE);
```

Comment:
A better practice is to set up width and length separately, and explicitly assign the same number to them. The reason to do so is to make maintenance easier if we need to update only one of them in the future, like
```
LENGTH = 50;
WIDTH = 50;
```
If we need them always to be equal to each other, we can add an extra check before this line. For example,
```
if (LENGTH != WIDTH) {
    throw EXCEPTION("image length and width must be the same.").
}
```